version.php

```diff
@@ -28,8 +28,8 @@
     die('Direct access to this script is forbidden.'); // It must be included from a Moodle page.
 }
 
-$plugin->version   = 2020051200;
-$plugin->release   = 2020051200;
+$plugin->version   = 2021073000;
```

Comment:
please change this to a micro bump ie 2020051201 because this is a supported but older branch